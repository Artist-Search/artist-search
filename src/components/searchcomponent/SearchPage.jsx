import React from 'react';

const SearchPage = () => {
  return (
    <div>
      {/* input, button and pagination on this page */}
      {/* <Pagination totalOffset={totalOffset}/> */}
      
    </div>
  );
};


export default SearchPage;

