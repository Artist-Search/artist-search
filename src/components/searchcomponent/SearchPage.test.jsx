import React from 'react';
import { shallow } from 'enzyme';
import SearchPage from './SearchPage';

describe('SearchPage component', () => {
  it('renders SearchPage', () => {
    const wrapper = shallow(<SearchPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
