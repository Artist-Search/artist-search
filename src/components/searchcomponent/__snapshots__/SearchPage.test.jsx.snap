// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPage component renders SearchPage 1`] = `
<Fragment>
  <SearchForm
    onChange={[Function]}
    onSubmit={[Function]}
    searchQuery=""
  />
  <Pagination
    count={0}
    nextPage={[Function]}
    prevPage={[Function]}
  />
  <ArtistSearchResults
    artists={Array []}
  />
</Fragment>
`;
